File: w03/d03/gadi_gottlieb/classwork/fibonacci/lib/fibonacci.rb
require 'pry'

def sum_of_even_fibonaccis(limit)
  holder = [1, 2]

  while (holder[-1] + holder[-2]) < limit
    holder << holder[-1] + holder[-2] # only concerned with the last one, and the second to last one.
  end                                 # we then add those two numbers together in order to get our Fibonacci numbers
  return holder.select { |num| num.even?}.reduce(:+)
end

binding.pry

File: w06/d05/charlye_tran/fibonacci.js
f1 = 1
f2 = 1
var numbers = [];
numbers.push(f1);
numbers.push(f2);
for (i = 0; numbers.length < 10; i++){
  numbers.push(f1+f2);
  f2 = f2 + f1
  f1 = f2 - f1
}
var total=0;
for(i = 0; i < numbers.length; i++) { total += numbers[i]; }
